sass/etna.scss

```diff
@@ -546,9 +546,44 @@ Modified to match the existing site (for now)
 // tna-useful-info component - 'Good to know' information from event page
 
 .tna-useful-info {
-    @include colour.light-accent-background;
-    color: colour.brand-colour("black");
-    padding: 1.5rem 2rem 2rem 2rem;
+    $useful-info-root: &;
+    display: flex;
+    flex-direction: column-reverse;
+    margin-top: 3rem;
+
+    @include media.on-larger-than-mobile {
+        align-items: flex-start;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+    }
+
+    &__content {
+        background-color: colour.brand-colour("light-grey");
+        color: colour.brand-colour("black");
```

Comment:
The text is slightly off-black - iirc correctly we used `@include colour.plain;` on another component

Reply:
@include colour.plain; didn't give the correct hex colour, so I have changed this to `color: $color__text;` (I couldn't see a suitable colour in `tna-frontend`)